index.js

```diff
@@ -11,6 +11,7 @@ const colors = {
   blue: '#7FDBFF',
   lightGray: '#DDDDDD',
   gray: '#AAAAAA',
+  orange: '#FF851B',
```

Comment:
In that case, you should not send a `console.groupCollapsed` call when there is an error or warning, and just throw `console.error` or `console.warn`.